pkgs/development/python-modules/duckduckgo-search/default.nix

```diff
@@ -26,23 +23,6 @@ let
       };
     }
   );
-
-  orjson_3_10_3 = orjson.overrideAttrs (
```

Comment:
Looks good, but it remains a problem in release-24.05. (https://github.com/NixOS/nixpkgs/pull/310576)
I would also like to remove the `curl-cffi` override and backport them.

Reply:
Good idea, job done.